pkg/util/uuid/uuid.go

```diff
@@ -1,5 +1,14 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package uuid provides a way to generate and cache a UUID for the host.
+// It leverages gopsutil for Linux and the machine GUID for Windows.
 package uuid
 
 import "github.com/DataDog/datadog-agent/pkg/util/cache"
 
+// GetUUID returns the UUID of the host
+var GetUUID = getUUID // For testing purposes
```

Comment:
**❓ question:** ‏I don't get why we need this (vs exporting functions in `pkg/util/uuid/uuid_*.go`) and more globally why `pkg/util/uuid` needs to be a module

Reply:
The `GetUUID` method would work on test runners and return their UUID, making tests using unstable data. Even if we could make them work, using stable data is safer to avoid flaky tests.

Thus, by exporting this var, we can replace the method everywhere in tests like we do [here](https://github.com/DataDog/datadog-agent/pull/22542/files#diff-8d1646ed07b96b27c7100f629ae5abf29929229b32fc2bcc7ee1c523d54ecc36R44-R51) .

Besides, it's a new package because we need to import it both in metadata payloads and in Remote Config. The Remote Config import was made in https://github.com/DataDog/datadog-agent/pull/22542/commits/4202275f8c720b6f163be460026795de81c5a099